src/threadpool.h

```diff
@@ -0,0 +1,92 @@
+inline void THROW_OR_ABORT(const std::string &s) {
```

Comment:
remove this function, turn the error below into an assert

Reply:
Ok I did that.